frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewList.tsx

```diff
@@ -46,18 +47,28 @@ const RevisionsOverviewListItem: React.FC<RevisionsOverviewListItemProps> = ({
 
 const RevisionsOverviewList: React.FC<RevisionsOverviewListProps> = ({ revisions, service }) => (
   <>
-    <SidebarSectionHeading text="Revisions" className="revision-overview-list">
-      {/* add extra check, if sidebar is opened for a knative deployment */}
-      {service.kind === ServiceModel.kind && (
-        <Button
-          variant="secondry"
-          onClick={() => trafficModalLauncher({ obj: service })}
-          disabled={!(revisions && revisions.length)}
-        >
-          Set Traffic Distribution
-        </Button>
-      )}
-    </SidebarSectionHeading>
+    <Flex
+      className="revision-overview-flex"
+      breakpointMods={
+        [{ modifier: 'justify-content-space-between' }, { modifier: 'align-items-center' }] as any
+      }
+    >
+      <FlexItem>
+        <SidebarSectionHeading text="Revisions" className="revision-overview-list" />
+      </FlexItem>
+      <FlexItem>
+        {/* add extra check, if sidebar is opened for a knative deployment */}
+        {service.kind === ServiceModel.kind && (
+          <Button
+            variant="secondary"
+            onClick={() => trafficModalLauncher({ obj: service })}
+            disabled={!(revisions && revisions.length)}
+          >
+            Set Traffic Distribution
+          </Button>
+        )}
+      </FlexItem>
+    </Flex>
```

Comment:
This is no longer needed because patternfly buttons define their font sizes. The old code should work. But fix the `variant` to be `secondary`. There's currently a spelling mistake.
You may need to adjust the flex alignment.

Reply:
This change is not required after switching to PF4 button.